src/Illuminate/Database/Eloquent/InvalidCastException.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Illuminate\Database\Eloquent;
+
+use RuntimeException;
+
+class InvalidCastException extends RuntimeException
+{
+    /**
+     * The name of the affected Eloquent model.
+     *
+     * @var object
```

Comment:
this should be `string`

Reply:
Are you sure that `$this->getModel()` in `HasAttributes` trait will return a string?

Reply:
No, but the call to get_class will...